src/webui/webapplication.cpp

```diff
@@ -172,6 +172,12 @@ void WebApplication::action_public_webui()
 
 void WebApplication::action_public_login()
 {
+    // return if session already exists or username/password auth isn't required
+    if (sessionActive() || (!isAuthNeeded() && sessionStart())) {
```

Comment:
I guess just `if (sessionActive())` sufficient to fix #6860 ?
I suspect session is already started at:
 https://github.com/qbittorrent/qBittorrent/blob/master/src/webui/abstractwebapplication.cpp#L125